app/src/main/java/org/thunderdog/challegram/navigation/ViewController.java

```diff
@@ -2248,7 +2250,8 @@ public static class Options {
     public final CharSequence info;
     public final OptionItem[] items;
 
-    public Options (CharSequence info, OptionItem[] items) {
+
+    public Options(CharSequence info, OptionItem[] items) {
```

Comment:
Thank you for the submission. Just leaving this comment to make sure you do not miss these links:

https://t.me/tgx_dev/1568
https://t.me/tgx_dev/1561

By quick jumping through the PR, I've noticed that there are several changes that could be excluded without affecting the result, like this one.

Note that they are insignificant and can be addressed after the deadline — meanwhile you may submit any commits up until the deadline if you have something more important in mind.

Reply:
Resolved, thanks!